vllm/engine/arg_utils.py

```diff
@@ -363,6 +364,10 @@ def add_cli_args(
             default=False,
             help='If True, the prefill requests can be chunked based on the '
             'max_num_batched_tokens')
+        parser.add_argument('--cpu-kvcache-space',
```

Comment:
QQ: Can we somehow make this an argument only for CPU backends?

Reply:
Right, this flag is too specific.

It is difficult to find a proper method using CLI arguments. I tried to change this parameter as an environment variable and highlight it using warning message when it is not set. Please look at the latest commit.